### PR TITLE
Adding BZ2120585 to 410

### PR DESCRIPTION
**Pull request by kquinn1204**
[BZ#2120585]: Intel E810 card unable to create a MACVLAN on interface already configured as SRIOV

Version(s):

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2120585
https://issues.redhat.com/browse/TELCODOCS-1115

Link to docs preview: Preview did not build

QE review: Merged to 4.12 in https://github.com/openshift/openshift-docs/pull/54573 needs merged in RN for 4.10.

QE has approved this change.
Additional information:

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2630,6 +2630,7 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 * Due to the inclusion of old images in some image indexes, running `oc adm catalog mirror` and `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
 
+* It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[BZ#2120585])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-10-release-notes.adoc:2633]: Not sure if it is MacVLan or macvlan?

**Review by JoeAldinger — COMMENTED** [review]
Small question about spelling that I wasn't sure about. Could you also upload a preview to the PR as well, please?

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-10-release-notes.adoc:2633]: @jeana-redhat I did a search on docs before adding to see what it is, and it seems to be macvlan across the docs.  I think we are OK with macvlan as I think it well understood this is the CNI plugin original content already merged to 4.12 RN so wold like to keep this the same.

Comment (stevsmit) [inline release_notes/ocp-4-10-release-notes.adoc:2633]: BZ needs to be in asterisks at the end.

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**